### DISCUSSION TITLE
Imposto de renda com acordo de bitributação

### DISCUSSION THREAD

**Discussion by f-leno** [DyegoCosta/trabalhando-remoto#195] [General]
Uma empresa está me oferecendo um trabalho home - office e estou tentando calcular quanto seria o salário mínimo para que faça sentido eu trocar de trabalho.

O que não estou conseguindo entender muito bem é como funcionaria o pagamento de imposto de renda, dado que a empresa opera em um país com acordo de bitributação (Rep. Tcheca).

Eu achei a tabela do carnê leão, mas caso algum imposto já seja retido na fonte como funcionaria? Eu seria cobrado o percentual total do imposto no Brasil em cima do líquido recebido na minha conta do Banco? Ou eu seria elegível a descontar um pouco do imposto que já paguei? Onde obter exatamente quais seriam as alíquotas, limites, etec.?

Alguém já passou por isso e poderia dar uma luz?

**Comment by diego-silva** [comment]
Como PF vais pagar o carnê leão que segue as mesmas faixas de trabalho assalariado "normal" do Brasil, com a diferença que tens de ser tu mesmo a pagar.

Normalmente não é retido nenhum imposto no país de origem (ao menos com as empresas que trabalhei de US e UK), mas se for, deves declarar com o bruto, e como o país tem acordo de bitributação seria só declarar esse valor retido num campo específico (Imposto Pago no Exterior a  Compensar) que ele abate do valor de imposto a pagar no BR (e pelo que lembro ainda acumula pros próximos meses se o valor retido for maior que o BR).
Podes fazer o download do programa do Carne Leão e simular com os teus valores para ver.

PS.: Se fores receber acima de uns 6k BRL provavelmente já compensa abrir uma empresa PJ.

**Comment by giuppep** [comment]
@diego-silva no caso do UK você teve que pagar o imposto duas vezes? Ou conseguiu pagar só a diferença?

**Comment by diego-silva** [comment]
@giuppep 
Por ser 'contractor' não pago imposto nenhum no UK, o dinheiro entra integral na conta… Depois pago os impostos no Brasil apenas.
Eu ACHO que UK tem acordo de bi-tributação com o BR, então se por acaso pagar imposto lá deve dar pra abater aqui…

**Comment by giuppep** [comment]
Obrigado pela resposta @diego-silva. Minha situação é um pouco diferente: sou um funcionario da empresa e recebo numa conta là. 

>  Eu ACHO que UK tem acordo de bi-tributação com o BR, então se por acaso pagar imposto lá deve dar pra abater aqui…

O UK não tem acordo com o Brasil, mas parece que o imposto pago no UK pode ser abatido aqui. Ainda não sei se isso de fato pode ser feito e como.

**Comment by diego-silva** [comment]
Pra fazer é fácil, no programa da receita em uma tab ali 'Rendimentos recebidos de PF ou do exterior (algo assim)' ali ele vai abrir uma tabelinha onde tens que jogar o valor bruto recebido na coluna 'Exterior' e depois tem a seção "Impostos pagos ou retidos" onde tu consegue lançar o imposto pago no exterior.

A questão maior é se pode lançar mesmo sem ter o acordo ou não, porque o programa provavelmente vai aceitar tudo, mas depois se forem fiscalizar pode dar ruim se não dava pra descontar, realmente não sei, mas deve explicar algo na ajuda do programa ou no 'perguntão'.

**Comment by diego-silva** [comment]
Achei aqui amigo:

> Pergunta 122 — Como são tributados os rendimentos de residente no Brasil recebidos do exterior na
existência de acordo internacional?
> …
> A compensação do imposto pago no exterior (desde que não seja compensado ou restituído no exterior) é
também possível no caso de o imposto ter sido pago em país cuja legislação permita a reciprocidade de
tratamento.
> …
> Não é necessária a prova de reciprocidade para a Alemanha, o Reino Unido e os Estados Unidos da América 

https://www.gov.br/receitafederal/pt-br/acesso-a-informacao/perguntas-frequentes/declaracoes/dirpf/pr-irpf-2021-v-1-1-2021-03-05.pdf

**Comment by giuppep** [comment]
muito obrigado @diego-silva! esse documento ajudou muito. Acabei de confirmar as informações com um amigo advogado tributarista. 

Mais tarde vou postar aqui os detalhes. Se quiser posso abrir uma PR para adicionar ao guia, pode ser util para outras pessoas também!
